common/Environments/Models/CreateComputeSystemOperation.cs

```diff
@@ -107,6 +109,11 @@ public void StartOperation()
         {
             try
             {
+                TelemetryFactory.Get<ITelemetry>().Log(
+                    "Environment_Creation_Event",
+                    LogLevel.Critical,
+                    new EnvironmentCreationEvent(ProviderDetails.ComputeSystemProvider.Id, EnvironmentsTelemetryStatus.Started));
```

Comment:
updated the ending for both classes to include UserEvent instead of just Event. I saw Davids message in the chat. Both of these are user-initiated events so the naming now lines up.